src/libsyncengine/update_detection/file_system_observer/remotefilesystemobserverworker.cpp

```diff
@@ -578,7 +584,7 @@ ExitCode RemoteFileSystemObserverWorker::extractActionInfo(const Poco::JSON::Obj
 ExitCode RemoteFileSystemObserverWorker::processAction(const SyncName &usedName, const ActionInfo &actionInfo,
                                                        std::set<NodeId, std::less<>> &movedItems) {
     SnapshotItem item(actionInfo.nodeId, actionInfo.parentNodeId, usedName, actionInfo.createdAt, actionInfo.modtime,
-                      actionInfo.type, actionInfo.size, actionInfo.canWrite);
+                      actionInfo.type, actionInfo.size, actionInfo.isLink, actionInfo.canWrite, true);
```

Comment:
And this is where the variable `isLink` was always set to `true` (at least set to the value of `canWrite`)

Reply:
Suggestion: define a constructor of `SnapshotItem` which task an object of type `ActionInfo` as argument.

Reply:
Done, but not exactly as suggested. Since `ActionInfo` only exist in `RemoteFileSystemObserverWorker` I have preferred to include a `SnapshotItem` object directly inside `ActionInfo`